.github/workflows/daily.yml

```diff
@@ -47,11 +48,11 @@ jobs:
         repository: ${{ env.GITHUB_REPOSITORY }}
         ref: ${{ env.GITHUB_HEAD_REF }}
     - name: make
-      run: make REDIS_CFLAGS='-Werror -DREDIS_TEST'
+      run: make SERVER_CFLAGS='-Werror -DSERVER_TEST'
```

Comment:
Let's add this renamed variable (not the CI but the Makefile change) to release notes. It's an interface change.